### ISSUE TITLE
Text over laps on safari with ?stringTest=double

### ISSUE THREAD

**Issue by orejolaphet** [phetsims/forces-and-motion-basics#156]
Test device: Personal Mac
Operating System: OSX 10.11
Browser: Safari 9.0
Problem description: 
Text for the Forces and masses check boxes over lap
Mass values overlap
PhET information overlaps

Screenshots:
<img width="337" alt="screen shot 2016-01-21 at 9 50 50 am" src="https://cloud.githubusercontent.com/assets/5631059/12487593/115aa7de-c025-11e5-878e-ae6f48979013.png">
<img width="504" alt="screen shot 2016-01-21 at 9 56 02 am" src="https://cloud.githubusercontent.com/assets/5631059/12487633/34b5790c-c025-11e5-8206-5886af1529b5.png">
<img width="353" alt="screen shot 2016-01-21 at 9 53 54 am" src="https://cloud.githubusercontent.com/assets/5631059/12487614/2109afea-c025-11e5-9212-5103b28e750d.png">

Similar to area builder issue https://github.com/phetsims/area-builder/issues/85

Troubleshooting information (do not edit):
Name: ‪Forces and Motion: Basics‬:‪Forces and Motion: Basics‬
URL: http://phettest.colorado.edu/forces-and-motion-basics/forces-and-motion-basics_en.html?ea&brand=phet&stringTest=double
Version: 1.1.5-dev.13 (require.js)
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11) AppleWebKit/601.1.56 (KHTML, like Gecko) Version/9.0 Safari/601.1.56
Language: en-us
Window: 1240x712
Pixel Ratio: 2/1
WebGL: WebGL 1.0 (2.1 INTEL-10.8.77)
GLSL: WebGL GLSL ES 1.0 (1.20)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 15 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 16)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}

Testing for Task https://github.com/phetsims/tasks/issues/479

**Comment by orejolaphet** [comment]
<img width="394" alt="screen shot 2016-01-21 at 10 07 15 am" src="https://cloud.githubusercontent.com/assets/5631059/12488002/c79174e6-c026-11e5-84bd-a3f7f7508f9e.png">

**Comment by jessegreenberg** [comment]
I think this is related to https://github.com/phetsims/scenery/issues/469, an issue where the strings overlap and are distorted on safari for the requireJS version of the sim.  In that issue, we decided not to address this since it did not affect built sims.

My guess is that this is going to come up every time we do testing out of phettest.  That might make it worth addressing.  Closing this issue for now.
